Examples/TicTacToe/tic-tac-toe/Sources/TwoFactorCore/TwoFactorCore.swift

```diff
@@ -57,6 +58,8 @@ public struct TwoFactor: Sendable {
         return .none
 
       case .view(.submitButtonTapped):
+        return .run { _ in await self.dismiss() }
```

Comment:
Old code?

Reply:
oops yes, just test code. reverted this and the stuff below.